docs/source/execution_device_interoperability.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>It might be nice to have a bit grabbier title here, as well as a more complete explanation of what this feature is. Maybe something like "Using cuML on CPU, GPU, or both" or something like that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuml/pull/5130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>